features/step_definitions/common_steps.rb

```diff
@@ -143,7 +145,7 @@
 end
 
 Then /^I am on #{MAYBE_VAR} page$/ do |page_name|
-  find('h1').text.should == normalize_whitespace(page_name)
+  page.should have_selector('h1', text: 'huh' + normalize_whitespace(page_name))
```

Comment:
Oops 😂 I was testing that it actually checks the text and forgot to change it back.